diff --git a/pybind/CMakeLists.txt b/pybind/CMakeLists.txt
index bdd41c3..ce6d65d 100644
--- a/pybind/CMakeLists.txt
+++ b/pybind/CMakeLists.txt
@@ -13,7 +13,10 @@ set_target_properties(pyposelib PROPERTIES OUTPUT_NAME _core)
 set_compile_flags(pyposelib)
 
 # Install the compiled module
-install(TARGETS pyposelib LIBRARY DESTINATION . COMPONENT python)
+install(
+    TARGETS pyposelib
+    LIBRARY DESTINATION ${Python_SITEARCH}/pyposelib COMPONENT python
+)
 
 if(GENERATE_STUBS)
     message(STATUS "Enabling stubs generation")
diff --git a/PoseLib/CMakeLists.txt b/PoseLib/CMakeLists.txt
index 8342f2f..1ecc2ab 100644
--- a/PoseLib/CMakeLists.txt
+++ b/PoseLib/CMakeLists.txt
@@ -111,3 +111,10 @@ include(${PROJECT_SOURCE_DIR}/cmake/LibraryConfig.cmake)
 
 # Eigen
 target_link_libraries(${LIBRARY_NAME} Eigen3::Eigen)
+
+install(
+    TARGETS ${LIBRARY_NAME}
+    LIBRARY DESTINATION lib
+    ARCHIVE DESTINATION lib
+    RUNTIME DESTINATION bin
+)
